### ISSUE TITLE
go get deprecated 

### ISSUE THREAD

**Issue by jelly-is-bean** [hashicorp/learn-terraform-functions#7]
and doesn't put the files in /home/terraform/go/src anymore - it puts it into /home/terraform/go/pkg/mod/github.com/hashicorp/learn-go-webapp-demo@version

This the code below i got to work:

```
#Create GOPATH for your user & download the webapp from github
sudo -H -i -u ${name} -- env bash << EOF
cd /home/${name}
export GOROOT=/usr/lib/go
export GOPATH=/home/${name}/go
export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
go get -d github.com/hashicorp/learn-go-webapp-demo
cd go/pkg/mod/github.com/hashicorp/learn-go-webapp-demo@v0.0.0-20200715024259-8f1f7d755351
go run webapp.go
EOF
```

Suggested alternative - use git:

```
# Create GOPATH for your user & download the webapp from github
sudo -H -i -u ${name} -- env bash << EOF
cd /home/${name}
export GOROOT=/usr/lib/go
export GOPATH=/home/${name}/go
export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
# these don't age well - using git instead
# go get -d github.com/hashicorp/learn-go-webapp-demo
# cd go/pkg/mod/github.com/hashicorp/learn-go-webapp-demo@v0.0.0-20200715024259-8f1f7d755351
git clone https://github.com/hashicorp/learn-go-webapp-demo.git
cd learn-go-webapp-demo
go run webapp.go
EOF
```

NOTE: thanks hashicorp for the tutorials - they are great

**Comment by lucstjean-ssc** [comment]
@jelly-is-bean Your alternative worked for me. Thanks!

**Comment by BrianMMcClain** [comment]
Thanks @jelly-is-bean! You're right, this tutorial needed an update since `go get` had been deprecated. This should be resolved as a part of #14. Thanks!
